src/o365/spo/commands/contenttype/contenttype-field-remove.ts

```diff
@@ -0,0 +1,307 @@
+      });
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.contentTypeId = (!(!args.options.contentTypeId)).toString();
+    telemetryProps.fieldLinkId = (!(!args.options.fieldLinkId)).toString();
+    return telemetryProps;
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-u, --webUrl <webUrl>',
+        description: 'Absolute URL of the site where the content type is located'
+      },
+      {
+        option: '-l, --listTitle <listTitle>',
+        description: 'Title of the list where the content type is located (if it is a list content type)'
+      },
+      {
+        option: '-i, --contentTypeId <id>',
+        description: 'The ID of the content type to process'
+      },
+      {
+        option: '-f, --fieldLinkId <id>',
+        description: 'The ID of the field to remove'
+      },
+      {
+        option: '-c, --updateChildContentTypes <updateChildContentTypes>',
+        description: 'Update child content types'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (!args.options.contentTypeId) {
+        return 'Required parameter contentTypeId missing';
+      }
+
+      if (!args.options.webUrl) {
+        return 'Required parameter webUrl missing';
+      }
+
+      if (!args.options.fieldLinkId) {
+        return 'Required parameter fieldLinkId missing';
+      }
+
+      if (!Utils.isValidGuid(args.options.fieldLinkId)) {
+        return `${args.options.fieldLinkId} is not a valid GUID`;
+      }
+
+      return true;
+    };
+  }
+
+  public commandHelp(args: {}, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(this.name).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} before using this command, log in to a SharePoint Online site using the
+      ${chalk.blue(commands.LOGIN)} command.
+        
+  Remarks:
+
+    To remove a field link from content type, you have to first log in to a SharePoint site using the ${chalk.blue(commands.LOGIN)} command,
+    eg. ${chalk.grey(`${config.delimiter} ${commands.LOGIN} https://contoso.sharepoint.com`)}.
+
+    If the specified ${chalk.grey('contentTypeID')} or ${chalk.grey('fieldLinkID')} doesn't refer to an existing objects, you will get
+    a an error.
+
+  Examples:
+  
+    Remove fieldLink with ID ${chalk.grey('2c1ba4c4-cd9b-4417-832f-92a34bc34b2a')} from content type with ID ${chalk.grey('0x0100CA0FA0F5DAEF784494B9C6020C3020A6')}
+    from web with Url ${chalk.grey('https://contoso.sharepoint.com')}
+      ${chalk.grey(config.delimiter)} ${this.name}  -i "0x0100CA0FA0F5DAEF784494B9C6020C3020A6" -f "880d2f46-fccb-43ca-9def-f88e722cef80" -u https://contoso.sharepoint.com
```

Comment:
In help examples, we should use the full names rather than short names (eg. `--contentTypeId` instead of `-i`) which are easier to read